### PR TITLE
Ron Jeffries

### PR DESCRIPTION
**Pull request by zaklinica**
Add info about Ron Jeffries

### CODE REVIEW

File: December/26/ron_jeffries/info.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+title: Ron Jeffries
+description: |
+    On December 24, 1939 was born Ron Jeffries.
+    He is an American software engineer and the creator of extreme programming.
+    Jeffries was one of the 17 original signatories of the Agile Manifesto.
+    Extreme and Agile methods are closely associated with Test-Driven Development (TDD).
+    He is co-author of the book 'Extreme Programming Installed' and
+    he co-edits the IEEE Software Special Issue on Test-Driven Development due in July/Aug 2007.
```
Comment (atodorov) [inline December/26/ron_jeffries/info.yaml:8]: Is there a reference for this IEEE magazine/article? The only one I see is https://ieeexplore.ieee.org/document/4163024 which is from the May/June issue. 

Maybe reword this entry to be less specific about the dates and more generic.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: December/26/ron_jeffries/info.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title: Ron Jeffries
+description: |
+    On December 24, 1939 was born Ron Jeffries.
```
[the remark this answers is not in the document]
Comment (zaklinica) [reply December/26/ron_jeffries/info.yaml:3]: ok

File: December/26/ron_jeffries/info.yaml (hunk 1 above)
Comment (zaklinica) [reply December/26/ron_jeffries/info.yaml:8]: ok
